src/mame/heathkit/tlb.cpp

```diff
@@ -1117,6 +1116,9 @@ void heath_superset_tlb_device::device_add_mconfig(machine_config &config)
 	m_maincpu->set_addrmap(AS_PROGRAM, &heath_superset_tlb_device::mem_map);
 	m_maincpu->set_addrmap(AS_IO, &heath_superset_tlb_device::io_map);
 
+	// per line updates are needed for onscreen menu to display properly
+	m_screen->scanline().set([](int scanline) {});
+
```

Comment:
Wouldn’t it make more sense to do `m_screen->set_video_attributes(VIDEO_UPDATE_SCANLINE);` if what you’re trying to do is to get the screen to do partial updates for each scan line?